portality/ui/messages.py

```diff
@@ -55,6 +55,8 @@ class Messages(object):
     EXCEPTION_ARTICLE_OVERRIDE = "Cannot update the article. An article with this URL and DOI already exists. If you are sure you want to replace it please delete it and then re-create it."
     EXCEPTION_SCRIPT_TAG_FOUND = "Metadata contains <script> tag. Please remove any <script> tags from your metadata and try again."
 
+    EXCEPTION_IDENTICAL_PISSN_AND_EISSN = "PISSN and EISSN are identical and equal {value}. They must be different"
```

Comment:
Suggest slightly alternative wording to match up with other uses of ISSN in the UI: `The Print and Online ISSNs supplied are identical. If you supply 2 ISSNs they must be different.`